research_daps/requirements.txt

```diff
@@ -0,0 +1,8 @@
+mysql-connector-python==8.0.22
+daps_utils @ git+https://github.com/nestauk/daps_utils@dev
```

Comment:
Should this be `dev`?

Reply:
You mean the bit before the @? That's just a label I think

Reply:
Sorry this was a note to myself while I was sorting some stuff out.

I was pointing out to myself that I shouldn't pin against `dev` because breaking changes could sneak in.